Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -34,7 +35,14 @@ private void OnAfterInteract(EntityUid uid, StickyComponent component, AfterInte
         if (args.Handled || !args.CanReach || args.Target == null)
             return;
 
-        // try stick object to a clicked target entity
+        //makes sure that neither the target nor the host have entities stuck to them or is stuck to entities
+        if (EntityManager.HasComponent<HasEntityStuckOnComponent>(uid)
+            || EntityManager.HasComponent<HasEntityStuckOnComponent>(args.Target)
+            || EntityManager.HasComponent<HasEntityStuckOnComponent>(uid)
+            || EntityManager.HasComponent<HasEntityStuckOnComponent>(args.Target))
```

Comment:
Is this supposed to be `IsStuckOnEntityComponent`? It's a duplicate of the above two lines right now.

Reply:
yeah it is a mistake , I ahve fixed it